judgments/tests/test_utils.py

```diff
@@ -81,3 +83,65 @@ def test_get_press_summaries_for_document_uri(self, mock_api_client):
 
         result = get_press_summaries_for_document_uri("sample_uri")
         self.assertEqual(result, summaries)
+
+    def test_show_no_exact_ncn_warning_is_true_when_no_exact_results_for_ncn_shaped_query(
```

Comment:
I think I'd prefer shorter, more cryptic function names and docstrings; there's a lot of negatives!